packages/valory/services/oracle_hardhat/service.yaml

```diff
@@ -113,17 +106,15 @@ type: skill
         api_id: protocollabs3
     params:
       args: *id001
-    server_api:
-      args: *id002
     benchmark_tool:
-      args: *id003
+      args: *id002
 ---
 public_id: valory/ledger:0.19.0
 type: connection
 config:
   ledger_apis:
     ethereum:
-      address: ${SERVICE_ORACLE_HARDHAT_RPC:str:http://hardhat:8545}
+      address: ${SERVICE_ORACLE_HARDHAT_RPC:str:http://host.docker.internal:8545}
```

Comment:
Will work after https://github.com/valory-xyz/open-autonomy/pull/1843

Reply:
No, this did not work.